events/Lambda/UpdateFunctionConfiguration20150331v2.json

```diff
@@ -9,6 +9,19 @@
     "mitreAttackTechniques": [
         "T1098 - Account Manipulation"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1059 - Command and Scripting Interpreter",
+            "reason": "Modifying Lambda function configurations allows execution of scripts or commands in the runtime environment."
+        },
+        {
+            "technique": "T1106 - Native API",
+            "reason": "Attackers use AWS API calls like UpdateFunctionConfiguration to directly interact with and manipulate Lambda functions to achieve their goals."
```

Comment:
This technique is only for OS application programming interface (API)

Reply:
Removed.